### PR TITLE
Update messaging rpc calls

### PR DESCRIPTION
**Pull request by blondfrogs**
- Allow the input of normal Root and Sub Asset name, and convert them to owner tokens
- Update rpc error messages

### CODE REVIEW

File: src/rpc/messages.cpp [hunk 1]
```diff
@@ -241,16 +249,25 @@ UniValue unsubscribefromchannel(const JSONRPCRequest& request) {
 
     AssetType type;
     if (!IsAssetNameValid(channel_name, type))
+        throw JSONRPCError(
+                RPC_INVALID_PARAMETER, "Channel Name is not valid after adding !.");
```
Comment (cfox) [inline src/rpc/messages.cpp]:
```suggestion
                RPC_INVALID_PARAMETER, "Channel Name is not valid.");
```

File: src/rpc/messages.cpp [hunk 2]
```diff
@@ -241,16 +249,25 @@ UniValue unsubscribefromchannel(const JSONRPCRequest& request) {
...
+
+    // if the given asset name is a root of sub asset, subscribe to that assets owner token
+    if (type == AssetType::ROOT || type == AssetType::SUB) {
+        channel_name += "!";
+        
+        if (!IsAssetNameValid(channel_name, type))
         throw JSONRPCError(
                 RPC_INVALID_PARAMETER, "Channel Name is not valid.");
```
Comment (cfox) [inline src/rpc/messages.cpp]:
```suggestion
                RPC_INVALID_PARAMETER, "Channel Name is not valid after adding !.");
```

File: src/rpc/messages.cpp (hunk 1 above)
Comment (blondfrogs) [reply src/rpc/messages.cpp]: Fixed

File: src/rpc/messages.cpp (hunk 2 above)
Comment (blondfrogs) [reply src/rpc/messages.cpp]: Fixed

**Pull request merged**